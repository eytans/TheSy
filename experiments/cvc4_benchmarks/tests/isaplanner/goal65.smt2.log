19:18:32 [WARN] Collecting statistics
19:18:32 [WARN]  ex is: zero
19:18:32 [WARN]  ex is: (succ zero)
19:18:32 [WARN]  ex is: (succ (succ zero))
19:18:32 [WARN]  ex is: nil
19:18:32 [WARN]  ex is: (cons autovar_Nat_0 nil)
19:18:32 [WARN]  ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:18:32 [WARN] Running exploration without proof mode, but goals were given
19:18:32 [WARN] Proof mode: true
19:18:32 [WARN]  ex is: zero
19:18:32 [WARN]  ex is: (succ zero)
19:18:32 [WARN]  ex is: (succ (succ zero))
19:18:32 [WARN]  ex is: nil
19:18:32 [WARN]  ex is: (cons autovar_Nat_0 nil)
19:18:32 [WARN]  ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:18:32 [WARN] Running exploration without proof mode, but goals were given
19:18:32 [WARN] Proof mode: true
19:18:32 [WARN] proved: (append ?ts_ph_Lst_0 nil) => ?ts_ph_Lst_0
19:18:32 [WARN] proved: (leq zero ?ts_ph_Nat_0) => true
19:18:43 [WARN] proved: (leq (succ zero) (succ ?ts_ph_Nat_0)) => true
19:18:43 [WARN] proved: (filter ?ts_ph_Lst_0) => (filter (filter ?ts_ph_Lst_0))
19:18:43 [WARN] proved: (filter (filter ?ts_ph_Lst_0)) => (filter ?ts_ph_Lst_0)
19:18:43 [WARN] proved: (append ?ts_ph_Lst_0 (append ?ts_ph_Lst_1 ?ts_ph_Lst_2)) => (append (append ?ts_ph_Lst_0 ?ts_ph_Lst_1) ?ts_ph_Lst_2)
19:18:43 [WARN] proved: (append (append ?ts_ph_Lst_0 ?ts_ph_Lst_1) ?ts_ph_Lst_2) => (append ?ts_ph_Lst_0 (append ?ts_ph_Lst_1 ?ts_ph_Lst_2))
19:18:43 [WARN] proved: (append (filter ?ts_ph_Lst_0) (filter ?ts_ph_Lst_1)) => (filter (append ?ts_ph_Lst_0 ?ts_ph_Lst_1))
19:18:43 [WARN] proved: (filter (append ?ts_ph_Lst_0 ?ts_ph_Lst_1)) => (append (filter ?ts_ph_Lst_0) (filter ?ts_ph_Lst_1))
