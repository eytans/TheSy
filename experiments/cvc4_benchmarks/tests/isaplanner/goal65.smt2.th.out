19:18:32 [0m[33m[WARN] [0mCollecting statistics
(leq ?__x0 ?__y1) => (or (= ?__x0 ?__y1) (less ?__x0 ?__y1))
(or (= ?__x0 ?__y1) (less ?__x0 ?__y1)) => (leq ?__x0 ?__y1)
(append nil ?x) => ?x
(append (cons ?x2 ?y) ?z) => (cons ?x2 (append ?y ?z))
(cons ?x2 (append ?y ?z)) => (append (cons ?x2 ?y) ?z)
(last (cons ?x3 ?y)) => (ite (= ?y nil) ?x3 (last ?y))
(ite (= ?y nil) ?x3 (last ?y)) => (last (cons ?x3 ?y))
(filter nil) => nil
nil => (filter nil)
(filter (cons ?x4 ?y)) => (ite (P ?x4) (cons ?x4 (filter ?y)) (filter ?y))
(ite (P ?x4) (cons ?x4 (filter ?y)) (filter ?y)) => (filter (cons ?x4 ?y))
19:18:32 [0m[33m[WARN] [0m ex is: zero
19:18:32 [0m[33m[WARN] [0m ex is: (succ zero)
19:18:32 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:18:32 [0m[33m[WARN] [0m ex is: nil
19:18:32 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_0 nil)
19:18:32 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:18:32 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:18:32 [0m[33m[WARN] [0mProof mode: true
19:18:32 [0m[33m[WARN] [0m ex is: zero
19:18:32 [0m[33m[WARN] [0m ex is: (succ zero)
19:18:32 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:18:32 [0m[33m[WARN] [0m ex is: nil
19:18:32 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_0 nil)
19:18:32 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:18:32 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:18:32 [0m[33m[WARN] [0mProof mode: true
19:18:32 [0m[33m[WARN] [0mproved: (append ?ts_ph_Lst_0 nil) => ?ts_ph_Lst_0
19:18:32 [0m[33m[WARN] [0mproved: (leq zero ?ts_ph_Nat_0) => true
19:18:43 [0m[33m[WARN] [0mproved: (leq (succ zero) (succ ?ts_ph_Nat_0)) => true
19:18:43 [0m[33m[WARN] [0mproved: (filter ?ts_ph_Lst_0) => (filter (filter ?ts_ph_Lst_0))
19:18:43 [0m[33m[WARN] [0mproved: (filter (filter ?ts_ph_Lst_0)) => (filter ?ts_ph_Lst_0)
19:18:43 [0m[33m[WARN] [0mproved: (append ?ts_ph_Lst_0 (append ?ts_ph_Lst_1 ?ts_ph_Lst_2)) => (append (append ?ts_ph_Lst_0 ?ts_ph_Lst_1) ?ts_ph_Lst_2)
19:18:43 [0m[33m[WARN] [0mproved: (append (append ?ts_ph_Lst_0 ?ts_ph_Lst_1) ?ts_ph_Lst_2) => (append ?ts_ph_Lst_0 (append ?ts_ph_Lst_1 ?ts_ph_Lst_2))
19:18:43 [0m[33m[WARN] [0mproved: (append (filter ?ts_ph_Lst_0) (filter ?ts_ph_Lst_1)) => (filter (append ?ts_ph_Lst_0 ?ts_ph_Lst_1))
19:18:43 [0m[33m[WARN] [0mproved: (filter (append ?ts_ph_Lst_0 ?ts_ph_Lst_1)) => (append (filter ?ts_ph_Lst_0) (filter ?ts_ph_Lst_1))
done in 11555
